odoo_backup_sh/__manifest__.py

```diff
@@ -7,7 +7,7 @@
     "category": "Backup",
     # "live_test_url": "",
     "images": ['images/backupsh.jpg'],
-    "version": "12.0.1.0.0",
+    "version": "12.0.1.0.1",
```

Comment:
No need to chage version as the module is not published

Reply:
Done